Gems/Multiplayer/Code/Source/Components/NetworkCharacterComponent.cpp

```diff
@@ -220,20 +220,23 @@ namespace Multiplayer
             Multiplayer::GetNetworkTime()->SyncEntitiesToRewindState(entitySweptBounds);
         }
 
-        if ((GetParent().m_physicsCharacter == nullptr) || (velocity.GetLengthSq() <= 0.0f))
+        if (GetParent().m_physicsCharacter != nullptr)
         {
-            return GetEntity()->GetTransform()->GetWorldTranslation();
+            GetParent().m_physicsCharacter->SetRotation(GetEntity()->GetTransform()->GetWorldRotationQuaternion());
+
+            if (velocity.GetLengthSq() > 0.0f)
+            {
+                GetParent().m_physicsCharacter->Move(velocity * deltaTime, deltaTime);
+                GetEntity()->GetTransform()->SetWorldTM(GetParent().m_physicsCharacter->GetTransform());
+                AZLOG(
```

Comment:
Log looks spammy, but surprised it’s always been there. I don’t recall seeing logs every time I TryMove. I guess leave it if it not a problem?

Reply:
It's a tagged log, you have to enable it. We have a whole bloom filter sitting outfront of the new log system that allows you to turn on and off logs at runtime through the console. Try running:
Logger.EnableLog NET_Movement
Works way better than putting in debug logs, commenting them out, deleting them, needing to add them back in next time you run into a problem, etc..